Makefile

```diff
@@ -126,6 +131,9 @@ docker-build: test ## Build docker image with the manager.
 docker-push: ## Push docker image with the manager.
 	docker push ${IMG}
 
+.PHONY: image
+image: docker-build docker-push
```

Comment:
I know this is sdk generated, but can we swap out the `docker-build` and `docker-push` for `image-build` `image-push`, and then use the container engine from above within each one?